### PR TITLE
Chrome blocks scripting into gallery by extension

### PR DESCRIPTION
**Pull request by pastak**
Gallery ( https://chrome.google.com/webstore/* ) is blacklisted by Google Chrome.

http://stackoverflow.com/questions/11613371/chrome-extension-content-script-on-https-chrome-google-com-webstore

### CODE REVIEW

**Comment by pastak** [comment]
Please check English message: `Sorry, this page cannot gyazo because blocked by Chrome. Please use native application.`

File: main.js [hunk 1]
```diff
@@ -146,6 +151,10 @@ chrome.browserAction.onClicked.addListener(function (tab) {
   chrome.tabs.insertCSS(tab.id, {
     file: './libs/menu.css'
   }, function () {
+    if (chrome.runtime.lastError && chrome.runtime.lastError.message.match(/cannot be scripted/)) {
+      window.alert('Sorry, this page cannot gyazo because blocked by Chrome. Please use native application.')
```
Comment (rakusai) [inline main.js]: Is is not allowed to use Gyazo extension in this page?

Comment (pastak) [inline main.js]: :+1:

**Comment by benfoden** [comment]
@pastak Here is my edit:
"Sorry, capturing with an extension is blocked on this page by Chrome. Please use the native Gyazo desktop application."

**Comment by pastak** [comment]
`Thanks for install. Let\'s gyazo on newtab!`

**Comment by pastak** [comment]
[Gyazo](https://gyazo.com/42814f1d474735038c3cacd2d0c16eb6)

**Comment by rakusai** [comment]
LGTM

**Pull request merged**